Frontend/src/Modal.svelte

```diff
@@ -16,7 +16,7 @@
 	<div on:click|stopPropagation>
 
 		<slot name="header" />
-		<hr />
+			<hr />
 		<slot />
 
 		<div class="modal-buttons">
```

Comment:
I can't comment on code from before this PR, but you should export a function for save, so you can provide that functionality to the dialog, so it does something before the dialog closes (LINE 26)

Reply:
Yes, I left it without it as I expect to add it in with the CRUD functionality. That said, it wouldn't hurt to just have the skeleton of a save function.

Reply:
That's totally fine if you want to wait till later! Just make sure that's captured in a story